server/utils/licenceFactory.js

```diff
@@ -50,6 +51,23 @@ function createConditionsObject(selectedConditions, formInputs) {
     }, {});
 }
 
+function createEligibilityObject(object) {
+    const acceptedAttributes = Object.keys(licenceModel.eligibility);
+
+    const filterIfUnwanted = {
```

Comment:
This name confused me a bit, sounds like a method. What about something like requiredSelectionForAttribute or something similar. Not easy to name!